### ISSUE TITLE
Wire that turns

### ISSUE THREAD

**Issue by josephcsible** [McJty/RFTools#1605]
The "Wire" can currently only transmit one redstone signal in a straight line. It should be able to turn left or right as well.

**Comment by Rixxaw** [comment]
Yeah maybe if you craft a machine frame + 2-4 wires together in a crafting grid you can "Form" the shape you want the Wire to be ...  Like Top Middle = Wire + Left Middle = Wire + Middle Middle = Machine Frame  gives you a Wire that has a right turn from Input to Output.  I'd have to think through the permutations, but if you had 4 Wires in your grid you would have to decide how many inputs / outputs you wanted.  Honestly I'm okay with the sides being either input or output, no reason i can think of for them to be directional.

**Comment by Shikifuyin** [comment]
Just want to bump this one ... the wire only going straight is a bit of a bummer ...
I'm surprised by the above over-complicated comment about crafting ...
Just have it configurable by a wrench or better by a GUI similar to all other blocks, where you can
just select which side(s) outputs a signal !
